### ISSUE TITLE
showing view count in two positions

### ISSUE THREAD

**Issue by MominRaza** [GATEOverflow/q2a-qnumbering#2]
![issu](https://user-images.githubusercontent.com/42776853/90115136-4f516a00-dd71-11ea-88a5-6b9f8134046c.png)

**Comment by arjunsuresh1987** [comment]
Is this behaviour different when not using the plugin?

**Comment by arjunsuresh** [comment]
Is this behaviour different without using this plugin?

**Comment by MominRaza** [comment]
Yes without this plugin view count only showing in on location (below votes count) but with this plugin view count showing in double position.(below votes count and below asked by)

**Comment by arjunsuresh** [comment]
is this issue still existing?

**Comment by shivasagarrao** [comment]
@arjunsuresh This issue is present in all the themes available on GO site.
